### PR TITLE
Desktop portfolio 2 Pull Request

### PR DESCRIPTION
**Pull request by adamteddychang**
Features Implemented

- Media query for Desktop compatible CSS
- Live Demo in the ReadMe

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -512,3 +525,253 @@ a {
 .social-icons li {
   padding: 8px;
 }
+
+/* Here is where the desktop css starts */
+@media (min-width: 600px) {
+  #menu-btn {
+    display: none;
+  }
```
Comment (uwadonat) [inline style.css:533]: - [ ] Min-width property on desktop should be in range of 992px - 1100px.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ My goal here is to create the first page of a mobile-friendly portfolio page.
 
 ## Live Demo
 ![screenshot](Images/Template_2_sample.PNG)
+(https://raw.githack.com/adamteddychang/Hello-Portfolio/Desktop-portfolio-2/index.html)
```
Comment (uwadonat) [inline README.md:15]: - [ ] Make sure your live link is pointing to the current work. Now it is broken.
Screenshot from 2021-09-23 20-22-54

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@adamteddychang**, 

Good job so far! :mechanical_arm: implementing most of the requirements for this project.

### To highlight! :+1: 
- Your pull request message is very descriptive ✔️
- Your Desktop version looks like the one in the design you chose  ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (adamteddychang) [reply style.css]: I have updated the min-width parameters.

File: README.md (hunk 2 above)
Comment (adamteddychang) [reply README.md]: Sorry about that, looking at the live demo link from my end is a little distorted but not as broken as the picture showed above. The screenshot from my end is shown below. Thank you for the catch. 
Live-demo
.

Comment (adamteddychang) [reply README.md]: This is how it looks like if the webpage is opened locally.
local-demo

**Review by uwadonat — APPROVED** [review]
Hi **@adamteddychang ** , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
cong1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**